src/main/java/at/hannibal2/skyhanni/config/features/misc/PersonOfInterestConfig.java

```diff
@@ -0,0 +1,36 @@
+package at.hannibal2.skyhanni.config.features.misc;
+
+import at.hannibal2.skyhanni.config.FeatureToggle;
+import com.google.gson.annotations.Expose;
+import io.github.notenoughupdates.moulconfig.annotations.ConfigEditorBoolean;
+import io.github.notenoughupdates.moulconfig.annotations.ConfigEditorText;
+import io.github.notenoughupdates.moulconfig.annotations.ConfigOption;
+
+public class PersonOfInterestConfig {
+
+    @Expose
+    @ConfigOption(name = "Enabled", desc = "Enable notification when a player from the list is in your lobby.")
+    @ConfigEditorBoolean
+    @FeatureToggle
+    public boolean enabled = false;
+
+    @Expose
+    @ConfigOption(name = "Players List", desc = "Players list you want to be notified for.\n§cCase sensitive, separated by comma.")
+    @ConfigEditorText
+    public String playersList = "hypixel";
+
+    @Expose
+    @ConfigOption(name = "Use Prefix", desc = "Should the [SkyHanni] prefix should be in the join/leave message ?")
+    @ConfigEditorBoolean
+    public boolean usePrefix = true;
+
+    @Expose
+    @ConfigOption(name = "Join Message", desc = "Configure the message when someone join.")
+    @ConfigEditorText
+    public String joinMessage = "&b%s &ajoined your lobby.";
```

Comment:
make it so that to use § you have to write &&, and also write both this and the fact that you have to use %s to show the playername in the config description